umbral/bignum.py

```diff
@@ -253,11 +254,16 @@ def __hash__(self):
 
 
 def hash_to_bn(crypto_items, params):
+    if not isinstance(crypto_items, list):
+        crypto_items = [crypto_items]
+
+    valid_instance_types = (BigNum, umbral.point.Point)
+
     blake2b = hashes.Hash(hashes.BLAKE2b(64), backend=backend)
     for item in crypto_items:
-        try:
+        if isinstance(item, valid_instance_types):
```

Comment:
The try-block is more pythonic here. Is there any reason why you are wanting to switch it?

Reply:
OK, I'll revert this